global.json

```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "version": "6.0.101"
```

Comment:
Did you add it to warn users about minimal supported version?
Then we need `rollForward` too with some less restrictive rule. Default `latestPatch` is too restrictive (6.0.200 would be discarded), it at least should be `latestFeature`

Reply:
Done